portal/server/helpers/constructPayload.js

```diff
@@ -5,16 +5,24 @@
  * @param {Object} body Request body (req.body)
  * @param {Array} properties Interested properties to be added into payload
  * @param {Boolean} csrf Include CSRF token, defaulted to `true`
+ * @param {Boolean} propertyEmpty List of property names that can't have empty value as they will have object value later, defaulted to True
  * @returns {Object} Payload
  */
-const constructPayload = (body, properties, csrf = true) => {
+const constructPayload = (body, properties, propertyEmpty, csrf = true) => {
```

Comment:
Because we are introducing new variable and this function was used before it is tempting to have default value for `propertyEmpty` variable.

Most current functions are passing just 2 variables, so `propertyEmpty` was set to empty (aka `false`) state. If we want to prevent all empty values and do global change, we could set default value to `true`.

Reply:
Agree set default value to `false`

```javascript
canPropertyEmpty = false
```

Reply:
Checked and default value now is false.